src/sources/audiosource.h

```diff
@@ -305,7 +305,7 @@ class AudioSource : public UrlResource, public virtual /*implements*/ IAudioSour
     explicit AudioSource(const QUrl& url);
 
     bool initChannelCountOnce(audio::ChannelCount channelCount);
-    bool initChannelCountOnce(SINT channelCount) {
+    bool initChannelCountOnce(int channelCount) {
```

Comment:
Unrelated: We should get rid of this overload and instead use ChannelCount::fromInt() explicitly where needed. If this causes too much noise then leave it as is.

Reply:
We need to keep the overload, to redirect calls to the checking int version.